Makd.mak

```diff
@@ -460,13 +460,18 @@ UNITTEST_FILES += $(call find_files,.d,,$C/$(SRC),$(TEST_FILTER_OUT))
 $O/fastunittests.d: $(filter-out %_slowtest.d,$(UNITTEST_FILES))
 $O/allunittests.d: $(UNITTEST_FILES)
 
-# 1. find any module named UnitTestRunner
-# 2. get the relative file path from `src` folder
-# 3. replace slashes with dots to get qualified module name
+
+# Default ocean test runner path
+OCEAN_TEST_RUNNER_MODULE=submodules/ocean/src/ocean/core/UnitTestRunner.d
```

Comment:
Mmm, it looks like you didn't have a look to my comment, this is not using git submodules to discover where/if ocean is used. I'm not saying that heuristic is flawless (it certainly isn't) but I think hardcoding the path like this is not a good idea, Makd doesn't impose any structure on where submodules should reside.